portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,58 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="author" content="Essence Cain">
+    <meta name="description" content="Essence Cain's portfolio page">
+    <meta name="keywords" content="Essence Cain, portfolio, NCAT">
+    <title>Essence Cain's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <ul class="navbar">
+          <li><a href="default.asp">Home</a></li>
+          <li><a href="about.asp">About Me</a></li>
+          <li><a href="contact.asp">Contact</a></li>
+      </ul>
+      <h1>Hello!</h1>
+      <h2>I am Essence Cain</h2>
+      <h3>About Me</h3>
+      <p>I am a current Computer Science student at North Carolina A&T State University, where I have began to explore mobile development and UX/UI design.</p>
+      <p>I believe technology will have a strong impact in advancing our society and I want to play a key role by offering a different perspective from undereppresnted groups 
+          in STEM fields.</p>
+      <h3>Work Experience</h3>
+      <img src="/images/google_icon.jpeg" alt="Google Logo">      
+      <dl>
+          <dt>Google</dt>
+          <dd>Student Training in Engineering Program Intern</dd>
+          <dd>May 2020 - Current</dd>
+      </dl>
+      <dl>
+          <dt>Google</dt>
+          <dd>Engineering Practicum Intern</dd>
+          <dd>May 2019 - August 2019</dd>
+      </dl>
+      <h3>Skills</h3>
+      <p>Java and C++</p>
+      <p>Mobile Development</p>
+      <p>UX/UI design</p>
+      <h3>Projects and Hackathons</h3>
+      <dl>
+          <dt>Be Smart Hackathon</dt>
+          <dd>Collaborated with a team of 3 students to design an application for American Airlines to improve customer service for passengers with disabilities. 
+              We develope our prototype in Figma to demo during the pitch.</dd>
+      </dl>
+      <h4>Let's Get In Touch</h4>
+      <a href="mailto: evcain1@gmail.com" target="_self">
+          <img src="/images/email_icon.jpeg" alt="Envelope Icon" style="border:0;">
+     </a>
+     <a href="https://www.linkedin.com/in/essence-cain" target="_self">
+          <img src="/images/linkedin_icon.jpeg" alt="LinkedIn Logo" style="border:0;">
+     </a>
+     <a href="https://github.com/essencec" target="_self">
+          <img src="/images/github.jpeg" alt="Github Logo" style="border:0;">
+     </a>
     </div>
   </body>
-</html>
+</html>
```

Comment:
You have have a line break at the end of this file and the css one. See the answer here for an explanation: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

You should also be able to configure your editor to automatically add one on save.